src/blob/index/benchmarks.rs

```diff
@@ -179,19 +176,14 @@ async fn benchmark_get_any() {
     const KEY_TO: u32 = 10_100_000;
     const PRINT_EVERY: u32 = 10_000;
 
-    let ioring = rio::new();
-    if ioring.is_err() {
-        println!("Current OS doesn't support AIO");
-    }
-    let ioring = ioring.ok();
     println!("Generating headers...");
     let headers = generate_headers(RECORDS_AMOUNT, KEY_MAPPER);
     let meta = generate_meta(META_SIZE);
 
     println!("Creating file index from headers...");
     let findex = FileIndexStruct::from_records(
         Path::new(FILEPATH),
-        ioring.clone(),
+        create_io_driver(),
```

Comment:
It's probably better to follow the general style: create an `IoDriver` at the beginning, store it in a local field, and pass its clone here